source/builtin_macros/src/attr_rewrite.rs

```diff
@@ -372,6 +374,22 @@ pub fn rewrite_verus_spec_on_fun_or_loop(
                 let unverified_fun = rewrite_unverified_func(&mut fun, with.with.span());
                 unverified_fun.to_tokens(&mut new_stream);
             }
+            if fun.sig.constness.is_some() {
+                let mut const_fun = fun.clone();
+                let span = fun.sig.constness.unwrap().span();
+                // It seems that we do not need to erase anything.
+                // But just do it to be safe and consistent with verus macro.
```

Comment:
Why do you not need to erase anything?

Reply:
ah yeah, that makes sense. The erasure will be more important after merging in https://github.com/verus-lang/verus/pull/1792 But for the time being, it shouldn't matter.